src/ucp/api/ucp.h

```diff
@@ -3275,8 +3274,10 @@ ucs_status_ptr_t ucp_put_nb(ucp_ep_h ep, const void *buffer, size_t length,
  *                                progress of the operation. The application is
  *                                responsible for releasing the handle using
  *                                @ref ucp_request_free "ucp_request_free()" routine.
+ * 
+ * @note The only datatype ucp_dt_make_contig(1) is supported.
```

Comment:
maybe add `in @a param->datatype, see @ref ucp_dt_make_contig` in the end?

Reply:
`Only the datatype ucp_dt_make_contig(1) is supported for @a param->datatype, see @ref ucp_dt_make_contig`

Reply:
reworded